docs/source/quickstart.rst

```diff
@@ -55,15 +55,29 @@ To build a model server docker image, you will also need to install
 `here <https://docs.docker.com/install/>`_.
 
 
-Hello World
------------
+Preface
+-------
 
-Before starting, let's prepare a trained model for serving with BentoML. Train a
+Before started, let's discuss how BentoML's project structure would look like. For most use-cases, users can follow this minimal scaffold
+for deploying with BentoML to avoid any potential errors (example project structure can be found under `guides/quick-start <https://github.com/bentoml/BentoML/tree/master/guides/quick-start>`_):
+
+.. code-block:: bash
+
+    bento_deploy/
+    ├── bento_packer.py       # responsible for packing BentoService
```

Comment:
usually `pack` will be part of the `train.py` script, right?  does it mean `train.py` will import `packer`?

Reply:
we could put pack as part of train scripts. We could separate the process if needed.